charts/keycloak-operator/crds/keycloak.org_keycloakbackups_crd.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
This is just a copy from the deploy/crds/ directory I assume, like the rest of the crd's here? It looks like there are no good ways of integrating them into the chart without duplication, but I'd like to here from Rich or Mohnish if it would be possible and/or preferred to leave them out of the chart completely and integrate them into the pipeline elsewhere to avoid the duplication. If not, I'd like to discuss with you either README updates that document that the files need to be in sync, deleting the copy from the deploy/crds directory, or updates to the make file to autogenerate these copies.